pkgs/development/libraries/portaudio/default.nix

```diff
@@ -1,11 +1,12 @@
 { lib, stdenv, fetchurl, alsa-lib, pkg-config, libjack2
 , AudioUnit, AudioToolbox, CoreAudio, CoreServices, Carbon }:
 
-stdenv.mkDerivation {
-  name = "portaudio-190600-20161030";
+stdenv.mkDerivation rec {
+  pname = "portaudio";
+  version = "190600-20161030";
 
   src = fetchurl {
-    url = "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz";
+    url = "http://www.portaudio.com/archives/pa_stable_v${version}.tgz";
```

Comment:
Replace dash with underscore in the URL.

Reply:
oh yes. very good that you found this little mistake. fixed it